lib/Dialect/FIRRTL/FIRRTLTypes.cpp

```diff
@@ -1148,6 +1152,7 @@ struct circt::firrtl::detail::FVectorTypeStorage
 
   /// This holds the bits for the type's recursive properties, and can hold a
   /// pointer to a passive version of the type.
+  RecursiveTypeProperties props;
```

Comment:
This is unused and ignored by, e.g., `FVectorType::getRecursiveTypeProperties`.

Reply:
Oof, good catch. Fixed with more extensive tests.